tools/tasks/render_workflow.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Set fact for colaltion path
+  ansible.builtin.set_fact:
+    collection_dir: "{{ collection_path + name }}"
+
+- name: Change to the collection directory
+  ansible.builtin.command:
+    cmd: "cd {{ collection_dir }}"
+    chdir: "{{ collection_dir }}"
+
+- name: Create a Git branch
+  ansible.builtin.command:
+    cmd: "git checkout -b gh_fix_{{ lookup('pipe', 'date +%m%d_%H%M%S') }}"
+    chdir: "{{ collection_dir }}"
+
+- name: Create .github/workflows if it is not present
+  ansible.builtin.file:
+    path: ".github/workflows"
+    state: directory
+
+- name: Copy the tests.yml template to the collection's gh workflow
+  ansible.builtin.template:
+    src: tests.yml.j2
+    dest: '{{ collection_dir + "/.github/workflows/tests.yml" }}'
+
+- name: Copy the linters.yml template to the collection's gh workflow
+  ansible.builtin.template:
+    src: linters.yml.j2
+    dest: '{{ collection_dir + "/.github/workflows/linters.yml" }}'
```

Comment:
Is there a need for either of these to be templated? It seems like we could just use `ansible.builtin.copy` here.